src/rules/selector-attribute-operator-space-after/index.js

```diff
@@ -0,0 +1,87 @@
+import selectorParser from "postcss-selector-parser"
+import {
+  report,
+  ruleMessages,
+  styleSearch,
+  validateOptions,
+  whitespaceChecker,
+} from "../../utils"
+
+export const ruleName = "selector-attribute-operator-space-after"
+
+export const messages = ruleMessages(ruleName, {
+  expectedAfter: (operator) => `Expected single space after "${operator}"`,
+  rejectedAfter: (operator) => `Unexpected whitespace after "${operator}"`,
+})
+
+export default function (expectation) {
+  return (root, result) => {
+    const checker = whitespaceChecker("space", expectation, messages)
+    const validOptions = validateOptions(result, ruleName, {
+      actual: expectation,
+      possible: [
+        "always",
+        "never",
+      ],
+    })
+    if (!validOptions) { return }
+
+    selectorAttributeOperatorSpaceChecker({
+      root,
+      result,
+      locationChecker: checker.after,
+      checkedRuleName: ruleName,
+      checkBeforeOperator: false,
+    })
+  }
+}
+
+export function selectorAttributeOperatorSpaceChecker({
+  locationChecker,
+  root,
+  result,
+  checkedRuleName,
+  checkBeforeOperator,
+}) {
+  root.walkRules(rule => {
+    if (rule.selector.indexOf("[") === -1
+      && rule.selector.indexOf("=") !== -1
```

Comment:
Should this be a `&&` or a `||`? Seems to me it could be a `||` but am I missing something?